src/module-elasticsuite-tracker/Model/EventIndex.php

```diff
@@ -76,6 +76,11 @@ public function indexEvents($events)
 
         foreach ($events as $event) {
             if (isset($event['page']['store_id'])) {
+                // Previous "date" column has been renamed to "created_at" in db_schema.xml
```

Comment:
missing a dot at the end, PHPcs will complain :)

Reply:
Yes, saw that. The only remaining diff between 2.10.x and 2.11.x tracker module is now a comment in the db_schema.xml